modules/op-event-pruner-reference.adoc

```diff
@@ -0,0 +1,76 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="event-pruner-reference_{context}"]
+= Configuration of the event-based pruner
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+You can configure the pruning behavior for the event-based pruner by modifying the `tekton-pruner-default-spec` ConfigMap in the `openshift-pipelines` namespace. This ConfigMap supports both global pruning policies and optional namespace-specific overrides.
+
+The following example shows a default `tekton-pruner-default-spec` ConfigMap that uses global pruning rules:
+
+.Example of the default `tekton-pruner-default-spec` ConfigMap
```

Comment:
Remove ".Example of the default `tekton-pruner-default-spec` ConfigMap" as per the new DITA guidelines.